src/consts.ts

```diff
@@ -8,10 +8,12 @@ export const COMBINER_PRETTY_NAMES: Readonly<Dictionary<string, SchemaCombinerNa
   [SchemaCombinerName.OneOf]: 'or',
 };
 
-export const NESTING_OFFSET: SpaceVals = 6;
-// @ts-expect-error: negative offset
-export const NEGATIVE_NESTING_OFFSET: NegativeSpaceVals = -NESTING_OFFSET;
+export const NESTING_OFFSET: SpaceVals = 3;
 
-export const CARET_OFFSET = NEGATIVE_NESTING_OFFSET;
-export const CARET_ICON_SIZE = 'xs';
-export const CARET_ICON_BOX_DIMENSION = 3;
+export const CARET_ICON_SIZE = 'sm';
+
+export const COMBINER_NAME_MAP: Dictionary<string, SchemaCombinerName> = {
```

Comment:
At first I was confused because I'd assume a `Dictionary` type's generics would be the key first and then the value, but apparently our `Dictionary` is backwards:
```
type Dictionary<T, K extends string | number = string> = { [key in K]: T };
```

So that makes sense. 😂